requirements.txt

```diff
@@ -6,3 +6,4 @@ pyyaml
 requests
 requests_cache
 schema
+urllib3==1.23
```

Comment:
Where were the uses of this added?

Reply:
Added a comment. This has to do with the requests_cache library and `ordered_dict`.